### PR TITLE
Lily/create accounts

### PR DESCRIPTION
**Pull request by lilylevrault**
Here's my new pull request, hopefully this one works better.

### CODE REVIEW

File: TA_Scheduler/views/account/create.py [hunk 1]
```diff
@@ -1,7 +1,45 @@
...
+                AccountUtil.createTAAccount(username, request.POST["password"])
+                return render(
+                    request,
+                    "account/create.html",
+                    {"message": "TA account '" + username + "' created"},
+                )
+            else:
+                return render(
+                    request, "account/create.html", {"message": "enter user type"}
+                )
```
Comment (schiltz3) [inline TA_Scheduler/views/account/create.py:45]: We don't use authority anymore, so I would recommend switching to using groups
   
```python
if request.user.is_anonymous:
    return redirect("/", {"error": "User is not authorized to create a course"})
elif request.user.groups.filter(name="admin).exists():
    #Something if admin
elif request.user.groups.filter(name="instructor").exists():
    #Something if instructor
elif request.user.groups.filter(name="ta").exists():
    #Something if ta
else:
   return redirect("/", {"error": "User is not authorized to create a course"})
```

Comment (lilylevrault) [reply TA_Scheduler/views/account/create.py:45]: usertype is something returned from the form (the type of user being created).  An if/elif/else statement like you suggest wouldn't really work as a replacement for the one I have in place.

Comment (schiltz3) [reply TA_Scheduler/views/account/create.py:45]: oh yeah mb. i read it wrong

**Review by schiltz3 — DISMISSED** [review]
Tests just need to pass

**Comment by lilylevrault** [comment]
Yeah, the checks that are failing seem to be just from tests not passing. I've shown a couple of people, and we haven't been able to figure out why it's happening.  The issue is simply that the context doesn't seem to contain <message>, despite the fact that <message> is a part of every rendering of account/create.  I've messed around with the site a bit though and all of the situations seem to be functional (including showing messages).

**Comment by schiltz3** [comment]
We may have to remove the test then. Rock made you run the tests when you present last semester, and he takes away a lot of points if tests don't pass

**Comment by lilylevrault** [comment]
Yikes. That'd only leave three tests for the page. Will he dock points for having too few?

**Comment by schiltz3** [comment]
I'm going over the test right now. I'll push all  the fixes in a bit

**Comment by schiltz3** [comment]
> 

yes

**Review by schiltz3 — APPROVED** [review]
Fixed it all

**Review by schiltz3 — APPROVED** [review]

**Review by llisaeva — APPROVED** [review]

**Pull request merged**